Serialization.Test/TestSerialization.fs

```diff
@@ -175,4 +175,87 @@ module RecordSerialization =
                 0xD8uy ||| 0uy; 0x02uy; // tag: fldnum=43, varint
                 0x01uy;                 // value true
             |]
+            
+    type Proto3Message  = {
+        id : int32
+        name : string
+        _unknownFields : RawField list
+        }
+        with
+            static member Default = {
+                id = 0
+                name = ""
+                _unknownFields = List.empty
```

Comment:
Should you remove this to show usage without unknown fields?  Or, comment this is optional?

Reply:
I'll add documentation as this is up the the code generator to enforce really.  There are a few parts where you can produce proto code that will fail at runtime like having no ring 0 for unknown fields, or forgetting to reverse fields in the fixup, so I think some docs will do for now.  A further cleanup in a future PR could tighten things up a little.